homeassistant/components/samsungtv/media_player.py

```diff
@@ -61,15 +61,15 @@
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the Samsung TV from a config entry."""
-    ip_address = config_entry.data[CONF_IP_ADDRESS]
+    host = config_entry.data[CONF_HOST]
```

Comment:
Migrate the old unique ids here to cover the case where the unique id changes from the `ip_address` to the `id`

Example:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/powerwall/__init__.py#L53

Reply:
Fixed